pkg/minikube/node/start.go

```diff
@@ -134,10 +134,14 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 		if err := kapi.ScaleDeployment(starter.Cfg.Name, meta.NamespaceSystem, kconst.CoreDNSDeploymentName, 1); err != nil {
 			klog.Errorf("Unable to scale down deployment %q in namespace %q to 1 replica: %v", kconst.CoreDNSDeploymentName, meta.NamespaceSystem, err)
 		}
-		// inject {"host.minikube.internal": hostIP} record into CoreDNS
-		if err := addCoreDNSEntry(starter.Runner, "host.minikube.internal", hostIP.String(), *starter.Cfg); err != nil {
-			klog.Errorf("Unable to inject {%q: %s} record into CoreDNS: %v", "host.minikube.internal", hostIP.String(), err)
-		}
+
+		// not running this in a Go func can result in DNS answering taking up to 38 seconds, with the Go func it takes 6-10 seconds
+		go func() {
+			// inject {"host.minikube.internal": hostIP} record into CoreDNS
+			if err := addCoreDNSEntry(starter.Runner, "host.minikube.internal", hostIP.String(), *starter.Cfg); err != nil {
+				klog.Errorf("Unable to inject {%q: %s} record into CoreDNS: %v", "host.minikube.internal", hostIP.String(), err)
```

Comment:
lets not spam the UI with raw error instead use 

```
klog.Warnning ....
```

and in addition to that a more user friendly UI error
out.Err("Failed to inject host.minikube.internal to coreDNS, this will limit the pods access to host IP")